tests/test_sql_query.py

```diff
@@ -599,42 +599,30 @@ async def test_sql_query_introspection_02(self):
             except Exception:
                 raise Exception(f'introspecting {table_name}')
 
-    @test.skip("unimplemented")
     async def test_sql_schemas(self):
-        await self.migrate(
-            '''
-            mod my_module {
-                type Foo;
-            }
-            '''
-        )
-        await self.squery('SELECT id FROM "my_module"."Foo";')
-        await self.squery('SELECT id FROM "public"."Person";')
-
-        await self.squery(
-            '''
-            SET search_path TO my_module, public;
-            SELECT id FROM "Foo";
-            '''
-        )
-        await self.squery(
-            '''
-            SET search_path TO my_module, public;
-            SELECT id FROM "Person";
-            '''
-        )
-        await self.squery(
-            '''
-            SET search_path TO public;
-            SELECT id FROM "Person";
-            '''
-        )
-        await self.squery(
-            '''
-            SET search_path TO my_module;
-            SELECT id FROM "Foo";
-            '''
-        )
+        await self.con.execute('''
+            create module my_module;
+            create type my_module::Foo;
+            create type default::Bar;
+        ''')
+        await self.con.query('SELECT my_module::Foo')
+        await self.scon.fetch('SELECT id FROM "my_module"."Foo";') # fails
+        await self.scon.fetch('SELECT id FROM "public"."Person";')
+        await self.scon.fetch('SELECT id FROM "default"."Person";')
+        await self.scon.fetch('SELECT id FROM "public"."Bar";') # fails
+        await self.scon.fetch('SELECT id FROM "default"."Bar";') # fails
```

Comment:
@fantix can you also look at this?

It behaves as it would not be using the new schema.

Reply:
This is probably because `TRANSACTION_ISOLATION` on the test case class is by default `True` and the new types are only valid within `self.con` but not `self.scon`.